components/SaveToDevice.js

```diff
@@ -111,34 +103,52 @@ componentDidMount() {
       // for andorid: externalDirectoryPath: /storage/emulated/0/Android/data/com.ruina/files
       // for android: externalStorageDirectoryPath: /storage/emulated/0
       // for ios: DocumentDirectoryPath: /var/mobile/Containers/Data/Application/12F7361A-BC3E-42C9-B81E-FBBBF7BA3E2C/Documents
-      const path_ios = RNFS.DocumentDirectoryPath + '/' + this.state.filename + "." + format;
-      const path_android = RNFS.ExternalStorageDirectoryPath + '/' + this.state.filename + "." + format;
+      var path_ios = RNFS.DocumentDirectoryPath + '/' + this.state.filename;
+      var path_android = RNFS.ExternalStorageDirectoryPath + '/' + this.state.filename;
```

Comment:
Does the filename automatically include the format now?

Reply:
Yes it does. This is not where the filneame is assigned though. In fact I believe these are not even used at all.